next/src/components/_article/hero/hero.module.scss

```diff
@@ -0,0 +1,40 @@
+.hero {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  padding: 50px 0px;
+  align-items: center;
+  gap: 24px;
+  background: var(--background);
+
+  header {
+    display: grid;
+    gap: 14px;
+
+    .categories {
+      display: flex;
+      flex-wrap: wrap;
+      gap: clamp(10px, calc(16vw/7.68), 24px);
+
+      .heading {
+        &:before {
```

Comment:
Dobrą praktyką jest używanie :: przed pseudo elementami zamiast :

Reply:
I agree. Look: https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-elements

Reply:
good to know